windows/guest_os_inplace_upgrade/handle_known_issue.yml

```diff
@@ -19,7 +19,7 @@
     - name: "Known issue - PVSCSI and VMXNET3 drivers not compatible with Windows Server after upgrade"
       ansible.builtin.debug:
         msg:
-          - "PVSCSI and VMXNET3 drivers installed by VMWare Tools 12.0.5 and later in Windows Server 2019 are not compatible with Windows Server 2022 and later after upgrade."
+          - "PVSCSI and VMXNET3 drivers installed by VMWare Tools (12.0.5 =< version < 12.2.0) in Windows Server 2019 are not compatible with Windows Server 2022 and later after upgrade."
```

Comment:
How about changing `(12.0.5 =< version < 12.2.0)` to `{{ vmtools_version }}`? So that it clearly shows the specific VMware Tools version has this issue.